pom.xml

```diff
@@ -1069,23 +1069,23 @@
             <configuration>
               <rules>
                 <requireJavaVersion>
-                  <version>1.6</version>
+                  <version>1.8</version>
                 </requireJavaVersion>
                 <requireMavenVersion>
                   <version>3.0.1</version>
                 </requireMavenVersion>
-                <!-- we can not use this as it require
-                compilation on 1.6 only, due to dependency to 'tools'
+                <!-- we can not use this as it require same version for all dependecies -->
+                <!--
                 <enforceBytecodeVersion>
-                  <maxJdkVersion>1.6</maxJdkVersion>
+                  <maxJdkVersion>1.8</maxJdkVersion>
                 </enforceBytecodeVersion>
                 -->
               </rules>
             </configuration>
           </execution>
         </executions>
-        <!-- we can not use this as it require
-        compilation on 1.6 only, due to dependency to 'tool'
+        <!-- required dependency for enforceBytecodeVersion  -->
```

Comment:
Why is it commented out if it is required?

Reply:
because enforceBytecodeVersion is commented out

Reply:
then we should keep the comment that says we can't use. that was removed and all thats left is "this commented out section is required".

Reply:
please recheck new wording.